specs/_features/eip7594/polynomial-commitments-sampling.md

```diff
@@ -484,6 +485,7 @@ def verify_cell_proof_batch(row_commitments_bytes: Sequence[Bytes48],
 
 ```python
 def construct_vanishing_polynomial(missing_cell_ids: Sequence[CellID]) -> Tuple[
+        Sequence[BLSFieldElement],
```

Comment:
The third parameter was not being used, which is likely why this did not trigger any runtime errors.

I can remove it in a subsequent PR (Did not want to make any changes that were not geared towards satisfying pylint)